lisa/tools/rpm.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
If it's not used, let's remove it.

Reply:
> If it's not used, let's remove it.

I didn't remove it because I thought there might be scenarios when we might need rpm tool itself. I would like to hear from others too on this or let me check more on whether we would need it for some old distros where we can't do with yum.
@glima - what do you think about this?

Reply:
Yeah, it can surface as needed one day. But keep it in a branch, till that day happens, I'd say :)

Reply:
> Yeah, it can surface as needed one day. But keep it in a branch, till that day happens, I'd say :)

Got it, removing from this PR.